napari/layers/image/image.py

```diff
@@ -405,7 +405,7 @@ def __init__(
     @property
     def _data_view(self):
         """Viewable image for the current slice. (compatibility)"""
-        return self._slice.image.view
+        return self._raw_to_displayed(self._slice.image)
```

Comment:
I'm also tempted to remove this, but we use it in the tests and it's been around for a while and might be something external users are depending on even though it's private. Maybe deprecate?

Reply:
Nah, we don't deprecate private things. LOL

Reply:
So in the end this is remaining?

Reply:
Good catch. In fact this is still being used in the vispy image layer, and thus the vispy labels layer, which is a little suspicious. Let me follow up and see if removing it (and maybe `raw_to_displayed` too?) is feasible here.